app/javascript/components/App.js

```diff
@@ -97,7 +98,7 @@ export default class App extends React.Component {
           <Route
             path="/borrow/:id"
             render={(props) => {
-              let book = findBook(this.state.books, props.match.params.id)
+              let book = this.findBook(this.state.books, props.match.params.id)
               return (
                 <BorrowShow book={book} />
               )
```

Comment:
Nice refactor, you can take this a step further by extracting all this into a function that will loop through some books that are passed in through the args and return back the array of `BorrowShow` books. 
You can see this section and the change made below are basically identical so you can just make one generic function that gets the data it needs.

Reply:
Hey Erik, we would love to talk to you about this. We would love to keep making the code better! We are going to take merge this so we can keep moving but we have this screenshot so we can talk about it on Monday!